lib/private/Setup.php

```diff
@@ -498,18 +498,18 @@ public static function updateHtaccess() {
 			$content .= "\n  RewriteRule ^core/js/oc.js$ index.php [PT,E=PATH_INFO:$1]";
 			$content .= "\n  RewriteRule ^core/preview.png$ index.php [PT,E=PATH_INFO:$1]";
 			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !\\.(css|js|svg|gif|png|html|ttf|woff|ico|jpg|jpeg|json|properties)$";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !core/img/favicon.ico$";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/robots.txt";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/remote.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/public.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/cron.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/core/ajax/update.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/status.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/ocs/v1.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/ocs/v2.php";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/updater/";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/ocs-provider/";
-			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !/ocm-provider/";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/core/img/favicon.ico$";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/robots.txt$";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/remote.php";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/public.php";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/cron.php";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/core/ajax/update.php";
+			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/status.php$";
```

Comment:
```suggestion
			$content .= "\n  RewriteCond %{REQUEST_FILENAME} !^/status\\.php$";
```

Also needing escaping to not machting by `status-php`?